db/migrations/20190919141224_create_ilk_state_history_table_and_triggers.sql

```diff
@@ -26,12 +26,14 @@ COMMENT ON COLUMN api.historical_ilk_state.ilk_identifier IS '@name id';
 CREATE FUNCTION ilk_rate_before_block(ilk_id INTEGER, header_id INTEGER) RETURNS NUMERIC AS
 $$
 WITH passed_block_number AS (
-    SELECT block_number FROM public.headers WHERE id = header_id
```

Comment:
In hindsight I probably should've put the formatting changes into a separate commit :-/